src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import {
+  useEffect,
+  useState,
+  FC,
+  ChangeEvent,
+  useCallback,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { CompletedStatus } from './types/CompletedStatus';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState<CompletedStatus>(CompletedStatus.All);
+  const [clickedTodoId, setClickedTodoId] = useState(0);
+  const [userTodo, setUserTodo] = useState<Todo | null>(null);
```

Comment:
```suggestion
  const [todo, setTodo] = useState<Todo | null>(null);
```

Reply:
done, but I should rename some variables which used in functions